### PR TITLE
Minor changes to Plane Curve and Curve Divisor

### PR DESCRIPTION
**Pull request by delphinepol**
Changed irreducibility function and use implemented colon ideal computation instead of "_qquotient"
in the computation of the global sections.

### CODE REVIEW

**Comment by fingolfin** [comment]
This needs a rebase now

File: test/Examples/PlaneCurve-test.jl [hunk 1]
```diff
@@ -114,9 +114,15 @@ end
...
 @testset "ProjPlaneCurve constructors" begin
+<<<<<<< HEAD
     R, (x, y, z) = PolynomialRing(QQ, ["x", "y", "z"])
     T, _ = grade(R)
     F = T(y^3 * x^6 - y^6 * x^2 * z)
+=======
+    R, (x,y,z) = PolynomialRing(QQ, ["x", "y", "z"])
+    T, _ = grade(R)
+    F = T(y^3*x^6 - y^6*x^2*z)
+>>>>>>> Minor changes to Plane Curve and Curve Divisor
```
Comment (fingolfin) [inline test/Examples/PlaneCurve-test.jl]: Here a merge conflict was not properly resolved

File: test/Examples/PlaneCurve-test.jl [hunk 2]
```diff
@@ -421,4 +427,4 @@ end
     p = Oscar.Pollard_p_1(n)
     @test gcd(n, m) == m
     @test gcd(n, p) == p
-end
+end
```
Comment (fingolfin) [inline test/Examples/PlaneCurve-test.jl]: Please add (resp. preserve) the trailing newline in this file :-)

File: test/Examples/PlaneCurve-test.jl (hunk 1 above)
Comment (delphinepol) [reply test/Examples/PlaneCurve-test.jl]: I probably forgot to save the file after removing these lines...

**Review by fingolfin — APPROVED** [review]

**Pull request merged**